targets/TARGET_STM/TARGET_STM32L4/pinmap.c

```diff
@@ -79,7 +79,17 @@ uint32_t Set_GPIO_Clock(uint32_t port_idx)
             gpio_add = GPIOE_BASE;
             __HAL_RCC_GPIOE_CLK_ENABLE();
             break;
-#endif
+#endif        
+        case PortF:
```

Comment:
Hello, can you please add #if defined (GPIOF_BASE) ? 
This file is located in STM32_L4 directory and shall be compatible with every STM32L4 device. STM32L432 for instance doesn't have GPIO port F or G.